docker/docker-compose.yml

```diff
@@ -330,7 +330,7 @@ services:
     container_name: supabase-analytics
     image: supabase/logflare:1.4.0
     healthcheck:
-      test: [ "CMD", "curl", "http://localhost:4000/health" ]
+      test: [ "CMD", "curl", "http://analytics:4000/health" ]
```

Comment:
This command is run from within container. So localhost should be correct. Same goes for other health checks.